### PR TITLE
Add updates after attendees feedback

### PR DESCRIPTION
**Pull request by perifaws**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/01-hpc-overview/01-updates.md [hunk 1]
```diff
@@ -8,3 +8,24 @@ weight: 20
 {{% notice info  %}}
 Frequently asked questions will be addressed on this page during the tutorial. Do not hesitate to visit this page often during the tutorial!
 {{% /notice %}}
+
+#### Accounts access
+
+Accounts took some time to be delivered and we are sorry that some of the attendees received them late in the tutorial. We will find a better mechanism next time to deliver the credentials in a timely manner.
```
Comment (sean-smith) [inline content/01-hpc-overview/01-updates.md:14]: Add "If you're still waiting for credentials please email us again at sc20tutorial@amazon.com."

File: content/01-hpc-overview/01-updates.md [hunk 2]
```diff
@@ -8,3 +8,24 @@ weight: 20
...
+
+
+#### You must specify a region issue
+
+Several attendees shared that they encountered the error below when creating the IAM policy for their cluster in [**Lab II**](/04-serverless/02-serverless-iam/02-iam-policy2.html). This was caused by a recent update and we apologies for the frustration this caused when running the lab. The responsible has been tasked in learning French and bake enough [Tarte au Framboises](http://nathaliebakes.com/tartes/tarte-aux-framboises-lenotre/) to feed the team after SC20.
```
Comment (sean-smith) [inline content/01-hpc-overview/01-updates.md:19]: "responsible" > "responsible party"

File: content/01-hpc-overview/01-updates.md [hunk 3]
```diff
@@ -8,3 +8,24 @@ weight: 20
...
+
+```
+aws cloudformation create-stack --stack-name pc-serverless-policy --parameters ParameterKey=S3Bucket,ParameterValue=serverless-${BUCKET_POSTFIX} --template-body file://serverless-template.yaml --capabilities CAPABILITY_NAMED_IAM, it replies : You must specify a region. You can also configure your region by running “aws configure”.
```
Comment (sean-smith) [inline content/01-hpc-overview/01-updates.md:22]: We need to add steps to fix it here

Comment (kraman-aws) [reply content/01-hpc-overview/01-updates.md:22]: i think there are two ways to fix this
#Install dependencies
sudo yum -y install jq
export AWS_REGION=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document | jq -r .region)

OR

export AWS_REGION=$(aws configure get region)

**Review by sean-smith — APPROVED** [review]

**Pull request merged**